Formula/agda.rb

```diff
@@ -109,17 +78,17 @@ def caveats
 
     if build.with? "stdlib"
       s += <<-EOS.undent
-      To use the Agda standard library, point Agda to the following include dir:
-        #{HOMEBREW_PREFIX}/lib/agda/src
-      EOS
+      The Agda standard library has been installed to:
+        #{HOMEBREW_PREFIX}/lib/agda
 
-      if build.with? "malonzo"
-        s += <<-EOS.undent
+      Agda 2.5.1 includes new library management features.
+      http://agda.readthedocs.org/en/stable/tools/package-system.html
 
-        To use the FFI bindings for the MAlonzo backend, give Agda the following option:
-          --ghc-flag=-package-db=#{HOMEBREW_PREFIX}/lib/agda/ffi/package.conf.d
-        EOS
-      end
+      To use the Agda standard library by default:
+        mkdir -p ~/.agda
+        echo #{HOMEBREW_PREFIX}/lib/agda/standard-library.agda-lib >>~/.agda/libraries
+        echo standard-library >>~/.agda/defaults
+      EOS
```

Comment:
The more text we have in the caveats the less likely any of it is read. As a result, we stick to providing things that only apply to Homebrew here (hence `caveats` rather than `docs`)